README.md

```diff
@@ -1,2 +1,41 @@
 # RunAway
-An infinity runner type Android game built on Unity.
+
+## Team members
+* Amartya Jayawant Patil
+* Devayani Shivankar
+* Vaishnavi Jarwal
+* Navya Sharma
+* Reshmika Sreenath Nambiar
+
+## Mentor
+* Shubhankar Gupta
+
+## Description
+RunAway is an endless runner Android game built on Unity. The player controls a mermaid who turns into a human on land. They have to dodge obstacles and collect coins to
```

Comment:
The mermaid scene hasn't been added yet, so mention that as a future scope, or an upcoming feature.

Reply:
I have put this part in the Future Scope.